src/main/java/io/github/amerebagatelle/fabricskyboxes/skyboxes/textured/TexturedSkybox.java

```diff
@@ -55,7 +57,7 @@ public final void render(WorldRendererAccess worldRendererAccess, MatrixStack ma
         RenderSystem.enableBlend();
 
         blend.applyBlendFunc();
-        RenderSystem.setShader(GameRenderer::getPositionTexColorShader);
+        RenderSystem.setShaderColor(1.0F, 1.0F, 1.0F, 1.0F); // Todo: Calculate our brightness based off position of the player and biomes/height ranges/weather and move this into Blend.
```

Comment:
What might this entail in the Blend object?  (what specifically is being moved in there)